website/docs/developer/api/api-gateway.md

```diff
@@ -0,0 +1,105 @@
+---
+id: API gateway paths, inputs and outputs
+title: API Gateway paths, inputs and outputs
+description: Additional notes on API gateway paths, inputs and expected outputs.
+---
+
+## Overview
+
+This documentation provides details on the paths, input and outputs for the `orca_api` API gateway. The format for the API invoke URL is `https://example.execute-api.us-west-2.amazonaws.com/[stage_name]/[path]`. Check [API gateway research webpage](https://github.com/nasa/cumulus-orca/blob/master/website/docs/developer/research/research-APIGateway.md) for more information.
+
+## Path 
+- The path `catalog/reconcile` of `orca_api` API gateway returns additional metadata information needed for reporting by triggering the `orca_catalog_reporting` lambda.
+- The path `recovery/granules` of `orca_api` API gateway returns detailed status of the granule by triggering the `request_status_for_granule` lambda.
+- The path `recovery/jobs` of `orca_api` API gateway returns detailed status for a particular recovery job by triggering the `request_status_for_job` lambda.
+
+### Inputs and outputs
+The API invoke URL can be found under `Stages` section in API gateway UI. All APIs use the `POST` method.
```

Comment:
End users may not be able to access the AWS UI. I would remove reference to it and stages. I would also move Inputs/Outputs under the specific path section so it is all together for a user.